.cmake-format.yaml

```diff
@@ -0,0 +1,7 @@
+# usage:
+# $ cmake-format -i CMakeLists.txt
```

Comment:
Could you add a Makefile target instead?  Also you could automate this in CI via `cmake-format --check`.

Reply:
OK, I appended a target in `CMakeLists.txt`.
Appending `cmake-format --check` in CI requires instration of `cmake-format` and its dependencies. So it should be done in another patch.